packages/svc-info/bin/index.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require('fs')
+
+const getGitInfo = () => {
+  const rev = fs
+    .readFileSync(`${process.cwd()}/.git/HEAD`)
+    .toString()
+    .trim()
+
+  const commitId = !rev.includes(':')
+    ? rev
+    : fs
+      .readFileSync(`${process.cwd()}/.git/` + rev.substring(5))
+      .toString()
+      .trim()
+
+  const repoName = fs
+    .readFileSync(`${process.cwd()}/.git/config`)
+    .toString()
+    .split('\n')
+    .filter((line) => /\turl =/.exec(line))
+    .reduce((acc, line) => acc + line.split('/').slice(-2).join('/'), '')
+
+  return {
+    commitId,
+    repoName,
+  }
+}
+
+const createBuildInfo = () => {
+  try {
+    const generateBuildInfo = process.BUILD_INFO || process.env.CI || process.env.TEAMCITY_VERSION
```

Comment:
1) this condition should be replaced with flag `--ci` flag.
2) `process.BUILD_INFO` does not look to be working.
3) output file location should be configurable. `--out=path`